microsoft-365/managed-desktop/service-description/privacy-personal-data.md

```diff
@@ -42,6 +42,10 @@ Microsoft Managed Desktop stores its data in one or more of the following Micros
 
 Microsoft Managed Desktop stores its data in the United States. Personal data is retained by Microsoft Managed Desktop for a maximum of 30 days.
 
+### Staff location
+
+The MMD Operations and MMD Security Operations teams are located in the United States and India.
```

Comment:
```suggestion
The Microsoft Managed Desktop Operations and Microsoft Managed Desktop Security Operations teams are located in the United States and India.
```

Reply:
"Microsoft Managed Desktop" has not been abbreviated anywhere else in this topic, so this suggestion is purely for consistency.

Reply:
@JohanFreelancer9 correct: Microsoft Managed Desktop must _never_ be abbreviated.